fulp_modules/features/antagonists/bloodsuckers/code/monster_hunter/monsterhunter_datum.dm

```diff
@@ -161,3 +200,75 @@
 	if(scan_target)
 		to_chat(owner, span_notice("You've lost the trail."))
 	. = ..()
+
+
+/datum/antagonist/monsterhunter/proc/insight_gained()
+	SIGNAL_HANDLER
+
+	var/description
+	var/datum/objective/assassinate/obj
+	if(objectives.len)
+		obj = pick(objectives)
+	if(obj)
+		var/datum/antagonist/heretic/heretic_target = IS_HERETIC(obj.target.current)
+		if(heretic_target)
+			description = "your target [heretic_target.owner.current.real_name] follows the [heretic_target.heretic_path], dear hunter."
+
+		else
+			description = "O' hunter, your target [obj.target.current.real_name] bears these lethal abilities:  "
+			for(var/datum/action/ability in obj.target.current.actions)
+				if(!ability)
+					continue
+				if(!istype(ability, /datum/action/changeling) && !istype(ability, /datum/action/bloodsucker))
+					continue
+				description += "[ability.name], "
+
+	rabbits_spotted++
+	to_chat(owner.current,span_notice("[description]"))
+
+/datum/antagonist/monsterhunter/proc/find_monster_targets()
+	var/list/possible_targets = list()
+	for(var/datum/antagonist/victim in GLOB.antagonists)
+		if(!victim.owner)
+			continue
+		if(victim.owner.current.stat == DEAD || victim.owner == owner)
+			continue
+		if(victim.owner.has_antag_datum(/datum/antagonist/changeling) || IS_BLOODSUCKER(victim.owner.current) || IS_HERETIC(victim.owner.current))
+			possible_targets += victim.owner
+
+	for(var/i in 1 to 3) //we get 3 targets
+		if(!(possible_targets.len))
+			break
+		var/datum/objective/assassinate/kill_monster = new
+		kill_monster.owner = owner
+		var/datum/mind/target = pick(possible_targets)
+		possible_targets -= target
+		kill_monster.target = target
+		kill_monster.update_explanation_text()
+		objectives += kill_monster
+
+
+/datum/antagonist/monsterhunter/proc/turn_beast()
+	SIGNAL_HANDLER
+
+	var/mob/living/simple_animal/hostile/megafauna/red_rabbit/evil_rabbit = new (get_turf(owner.current))
+	owner.current.gib()
+	owner.transfer_to(evil_rabbit)
+	var/datum/objective/survive/destruction = new
+	destruction.name = "Wreak Havoc"
+	destruction.explanation_text = "Wreak havoc upon the station"
+	destruction.owner = owner
+	objectives += destruction
+	for(var/obj/machinery/power/apc/apc as anything in GLOB.apcs_list)
```

Comment:
Do we want EVERY single apc to overload? I don't think this happens at any other time in the game, even in lategame cult nar'sie-level shit. Seems more annoying than anything, and doesn't make much sense either.

Reply:
ma